rye/src/cli/publish.rs

```diff
@@ -42,6 +42,9 @@ pub struct Args {
     /// Path to alternate CA bundle.
     #[arg(long)]
     cert: Option<PathBuf>,
+    /// Continue uploading files if one already exists (only applies to repositories supporting this feature)
```

Comment:
Might be that this is what originally threw me off (I used part of Twine's description of this flag). It might be better worded as "Skip uploading any files that have already been published." But I'm not sure if it's just a me problem.

Reply:
PR'd it anyway #832